packages/pf4-component-mapper/src/files/dual-list-select.js

```diff
@@ -53,11 +57,11 @@ const List = ({ value, optionClick, noOptionsTitle, filterValue, filterValueText
           onClick={(e) => optionClick(e, value)}
           key={value}
           value={value}
+          {...ListItemProps}
+          {...ListItemPropsItem}
           className={`ddorg__pf4-component-mapper__dual-list-select-option ${
             selectedValues.includes(value) ? 'ddorg__pf4-component-mapper__dual-list-select-option-selected' : ''
```

Comment:
You should convert this into clsx function and add `ListItemProps.className` and `ListItemPropsItem.className`